documentation/source/getting_started/components_architecture.rst

```diff
@@ -0,0 +1,23 @@
+Components and Architecture
+===========================
+This section includes OpenEBS components and architecture.
+
+Components
+----------
+
+OpenEBS platform contains three major components:
+
+  * Storage PODs or VSMs
```

Comment:
We just wanted to say Storage Containers

Reply:
Done! Changes will reflect when I push the document to the readthedocs website.